build/release/cdn.js

```diff
@@ -1,130 +1,162 @@
+	"jquery.module.min.map": "dist-module/jquery.module.min.map",
+	"jquery.module.slim.js": "dist-module/jquery.module.slim.js",
+	"jquery.module.slim.min.js": "dist-module/jquery.module.slim.min.js",
+	"jquery.module.slim.min.map": "dist-module/jquery.module.slim.min.map"
+};
 
 /**
  * Generates copies for the CDNs
  */
 function makeReleaseCopies( Release ) {
-	shell.mkdir( "-p", cdnFolder );
-
-	Object.keys( releaseFiles ).forEach( function( key ) {
-		var text,
-			builtFile = releaseFiles[ key ],
-			unpathedFile = key.replace( /VER/g, Release.newVersion ),
-			releaseFile = cdnFolder + "/" + unpathedFile;
-
-		if ( /\.map$/.test( releaseFile ) ) {
-
-			// Map files need to reference the new uncompressed name;
-			// assume that all files reside in the same directory.
-			// "file":"jquery.min.js" ... "sources":["jquery.js"]
-			text = fs.readFileSync( builtFile, "utf8" )
-				.replace( /"file":"([^"]+)"/,
-					"\"file\":\"" + unpathedFile.replace( /\.min\.map/, ".min.js\"" ) )
-				.replace( /"sources":\["([^"]+)"\]/,
-					"\"sources\":[\"" + unpathedFile.replace( /\.min\.map/, ".js" ) + "\"]" );
-			fs.writeFileSync( releaseFile, text );
-		} else if ( builtFile !== releaseFile ) {
-			shell.cp( "-f", builtFile, releaseFile );
-		}
+	[
+		{ filesMap: versionedReleaseFiles, cdnFolder: cdnFolderVersioned },
+		{ filesMap: unversionedReleaseFiles, cdnFolder: cdnFolderUnversioned }
+	].forEach( ( { filesMap, cdnFolder } ) => {
+		shell.mkdir( "-p", cdnFolder );
+
+		Object.keys( filesMap ).forEach( function( key ) {
+			var text,
+				builtFile = filesMap[ key ],
+				unpathedFile = key.replace( /@VER/g, Release.newVersion ),
+				releaseFile = cdnFolder + "/" + unpathedFile;
+
+			if ( /\.map$/.test( releaseFile ) ) {
+
+				// Map files need to reference the new uncompressed name;
+				// assume that all files reside in the same directory.
+				// "file":"jquery.min.js" ... "sources":["jquery.js"]
+				text = fs.readFileSync( builtFile, "utf8" )
+					.replace( /"file":"([^"]+)"/,
+						"\"file\":\"" + unpathedFile.replace( /\.min\.map/, ".min.js\"" ) )
+					.replace( /"sources":\["([^"]+)"\]/,
+						"\"sources\":[\"" + unpathedFile.replace( /\.min\.map/, ".js" ) + "\"]" );
+				fs.writeFileSync( releaseFile, text );
+			} else if ( builtFile !== releaseFile ) {
+				shell.cp( "-f", builtFile, releaseFile );
+			}
+		} );
+
 	} );
 }
 
-function makeArchives( Release, callback ) {
+async function makeArchives( Release ) {
 
 	Release.chdir( Release.dir.repo );
 
-	function makeArchive( cdn, files, callback ) {
-		if ( Release.preRelease ) {
-			console.log( "Skipping archive creation for " + cdn + "; this is a beta release." );
-			callback();
-			return;
-		}
+	async function makeArchive( { cdn, filesMap, cdnFolder } ) {
+		return new Promise( ( resolve, reject ) => {
+			if ( Release.preRelease ) {
+				console.log( "Skipping archive creation for " + cdn + "; this is a beta release." );
+				return;
```

Comment:
This needs to resolve right?

Reply:
Nice catch. I haven't tested the beta flow. Fixed.